migration/idempotent/001-base.sql

```diff
@@ -2017,6 +2086,7 @@ GRANT EXECUTE ON FUNCTION prom_api.is_normal_nan(double precision) TO prom_reade
 CREATE OR REPLACE FUNCTION prom_api.val(
         label_id INT)
     RETURNS TEXT
+    SET search_path = pg_catalog
```

Comment:
we should probably remove this and make it inlinable

Reply:
Same as above, I'm not sure that this is inlinable in its current form.